grobid-trainer/resources/dataset/citation/corpus/crossref_raw_citations.xml

```diff
@@ -0,0 +1,54 @@
+<bibl> <author>Christoph Molnar</author>. <date>2020</date>. <title level="m">Interpretable Machine Learning</title> .<publisher>Leanpub</publisher> <pubPlace>Victoria Canada</pubPlace>. <ptr type="web">https://christophm.github.io/interpretable-ml-book/</ptr>. </bibl>
+<bibl> <author>Günter Klambauer Thomas Unterthiner Andreas Mayr and Sepp Hochreiter</author>. <date>2017</date>. <title level="a">Self-Normalizing Neural Networks</title>. <idno type="arXiv">arxiv: 1706.02515</idno> [cs.LG] </bibl>
+<bibl> <author>Byung-Hak Kim Ethan Vizitei and Varun Ganapathi</author>. <date>2018</date>. <title level="a">GritNet: Student Performance Prediction with Deep Learning</title>. <idno type="arXiv">arxiv: 1804.07405</idno> [cs.LG] </bibl>
+<bibl> <author>Sercan O. Arik and Tomas Pfister</author>. <date>2019</date>. <title level="a">TabNet: Attentive Interpretable Tabular Learning</title>. <idno type="arXiv">arxiv: 1908.07442</idno> [cs.LG] </bibl>
+<bibl> <orgName>Wikipedia</orgName>. <date>2019</date>. <title level="m">PageRank</title>, Retrieved <date>September 5, 2019</date>, from <ptr type="web">https://en.wikipedia.org/wiki/PageRank</ptr> </bibl>
+<bibl> <author>Mohammad Al Hasan, Vineet Chaoji, Saeed Salem, and Mohammed Zaki</author>. [n.d.]. <title level="m">Link Prediction using Supervised Learning</title>. ([n. d.]), 10. </bibl>
+<bibl> <author>Johan J Vossensteyn Andrea Kottmann Benjamin WA Jongbloed Franciscus Kaiser Leon Cremonini Bjorn Stensaker Elisabeth Hovdhaugen and Sabine Wollscheid</author>. <date>2015</date>. <title level="m">Dropout and completion in higher education in Europe: Main report</title>. <note type="report">Technical Report</note>. <orgName>CHEPS</orgName> and <orgName>NIFU</orgName> . </bibl>
+<bibl> <author>Xue Bin Peng, Marcin Andrychowicz, Wojciech Zaremba, and Pieter Abbeel</author>. <date>2018</date>. <title level="a">Sim-to-Real Transfer of Robotic Control with Dynamics Randomization</title>. <title leve="m">Proceedings - IEEE International Conference on Robotics and Automation</title> (<date>2018</date>), <biblStruct type="pp">3803--3810</biblStruct>. </bibl>
+<bibl> <author>Max Pumperla</author>. <date>2019</date>. <title level="m">Hyperas</title>. <ptr type="web">https://github.com/maxpumperla/hyperas</ptr> . </bibl>
+<bibl> <author>Oktay K, Karlikaya G, Aydin BA</author>. <title level="a">Ovarian transplantation now a reality?</title> In: <title level="m">International Symposium on Storing Reproduction</title>, <date>1999</date>; <pubPlace>Bologna</pubPlace>. p. <biblStruct type="pp">O23</biblStruct>. </bibl>
```

Comment:
`<biblStruct type="pp">O23</biblStruct>` -> `<biblScope unit="page">O23</biblScope>`

interesting, the `O` in `O23` is actually a letter not the digit !

Reply:
Yeah, here is an example with that citation: https://www.ejog.org/article/S0301-2115(00)00479-6/references

I think it is a typo or OCR error, other articles cite it as page 23. But that is the exact citation string from the publisher.